components/app-core/backend/repository/console_config/psql_console_config.go

```diff
@@ -125,7 +125,19 @@ func (c *ConsoleConfigRepository) IsInitialised() (bool, error) {
 
 	rowCount, err := c.getTableCount()
 	if err != nil {
-		return false, err
+		for strings.Contains(err.Error(), "does not exist") {
+			// Schema isn't initialised yet. Wait a few secs and retry
+			log.Warnf("It appears schema isn't initialised yet, sleeping and trying again %s", err)
+			time.Sleep(1 * time.Second)
+			rowCount, err = c.getTableCount()
+			if err == nil {
+				break
+			}
+		}
+		if err != nil {
```

Comment:
This can be true.
There are two paths here:
1. In one case, when the error is actually due to the schema not being initialised, we will end up waiting in the for loop until it is initialised and err in that case will indeed be nil.
2. The other case is, if the error is caused by something different, the for loop will never execute and conditional will be true.